app/test_main.py

```diff
@@ -1,3 +1,57 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_value",
+    [
+        pytest.param(
+            0, 0,
+            [0, 0],
+            id="should return zero when dog-cat ages equal 0"
+        ),
+        pytest.param(
+            14, 14,
+            [0, 0],
+            id="should return zero when dog-cat ages less 14"
+        ),
```

Comment:
This test case seems to be incorrect. If the dog or cat age is 14, it should not return [0, 0]. According to the common rule, the first 2 years of a cat or dog's life are roughly equal to 24 human years, and after that, each additional year is approximately 4 human years for a dog and 5 for a cat. Please adjust your test case accordingly.